### ISSUE TITLE
Path to serveradmin in service_check incorrect on Mac OS X 10.6 Server

### ISSUE THREAD

**Issue by Steffan-Ravn** [munkireport/munkireport-php#480]
The service_check script inside the service-module only points to the correct path for serveradmin installed via Mac App Store, as Snow Leopard Server was purchased as a separate OS, the service_check script fails on 10.6 Server.

These lines in service_check:
```
# Serveradmin executable
SERVERADMIN="/Applications/Server.app/Contents/ServerRoot/usr/sbin/serveradmin"
```

Should be change to this:
```
# OS check
OSVERSION=$(/usr/bin/sw_vers -productVersion | /usr/bin/cut -d . -f 2)

# Serveradmin executable
if [[ ${OSVERSION} -lt 8 ]]; then
    SERVERADMIN="/usr/sbin/serveradmin"
else
    SERVERADMIN="/Applications/Server.app/Contents/ServerRoot/usr/sbin/serveradmin"
fi
```

That will ensure data-collection on both 10.6 or 10.7 Server and the newer MAS-version of OS X Server.

**Edit:** As pointed out by @MichalMMac it's also different on Lion Server, so I've modified my corrections to support both 10.6 and 10.7 Server.

**Comment by MichalMMac** [comment]
`/usr/sbin/serveradmin` is also applicable for 10.7 Lion with OS X Server.

**Comment by Steffan-Ravn** [comment]
I did not know that - but as Lion also has the MAS-version, wouldn't it still work on Lion with /Applications/Server.app/Contents/ServerRoot/usr/sbin/serveradmin ?

**Comment by MichalMMac** [comment]
Nope. There is no ServerRoot in Lion Server.app. App itself weight about 40+ MB.

 Lion Server was kind of a hybrid between old Mac OS X Server world and new. There was still ServerAdmin present for most of serious management.

**Comment by Steffan-Ravn** [comment]
Thanks for the info - I have no Lion Server handy. :)
I've updated my corrections, so both 10.6 and 10.7 Server should be good to go.

**Comment by bochoven** [comment]
Could you create a pull request with your proposed changes?

-Arjen

**Comment by Steffan-Ravn** [comment]
Done. :)

**Comment by bochoven** [comment]
Resolved with #483
